torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -188,13 +193,13 @@ void setstateTuple(
   if (checkHasValidSetGetState(type)) {
     Function& setstate = type->getMethod("__setstate__");
     if (setstate.isGraphFunction()) {
-      elements.push_back(
-          getFunctionTuple(module, setstate, save_debug_info_in_bytecode));
+      getFunctionTuple(
+          module, setstate, elements, debug_info_elements, save_debug_info_in_bytecode);
```

Comment:
Let's function return pair. You can capture the return by reference but most likely they should be return value optimized even without capturing by ref. https://en.cppreference.com/w/cpp/language/copy_elision